CONTRIBUTING.md

```diff
@@ -74,6 +74,24 @@ You can use IntelliJ IDEA (preferred) or VS Code (alternate) in the browser.
 If you prefer using IntelliJ IDEA, you can setup Gitpod integration with JetBrains Gateway using the instructions on [gitpod.io](https://www.gitpod.io/docs/ides-and-editors/intellij),
 which will open the workspace in IntelliJ IDEA using JetBrains Gateway.
 
+### Linting
+
+For linting we use a number of tools:
```

Comment:
What about Checkstyle? This has no automatic fix right?

Reply:
I forgot that one. I tried to cover it with "To automatically fix most issues run"

A number of eslint and stylelint issues are not automatically fixable (although many are)